app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -121,12 +118,13 @@ public void onDrawerStateChanged(int newState){
             selectItem(0);
         }
 
-        checkInformedConsent();
-    }
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        if (!preferences.getBoolean("cleanup_unused_files", false)) {
+            TestStorage.removeUnusedFiles(this);
+            PreferenceManager.getDefaultSharedPreferences(this).edit().putBoolean("cleanup_unused_files", true).apply();
+        }
```

Comment:
I imagine this is the cleanup code for the issue with files not really being deleted when you deleted them on Android?

If so the logic is to run it only once for the first time, correct? What is the plan for the future, like are we from now on always going to be checking for this?